lib/app/Auth/PhoenixTransactionBridge.php

```diff
@@ -22,6 +22,12 @@ public function getHeader($name) {
    * @return void
    */
   public function log($message, array $details) {
-    watchdog('PhoenixTransactionBridge', $message, $details, WATCHDOG_INFO);
+    $details = implode(', ', array_map(
+        function ($value, $key) { return sprintf("%s='%s'", $key, $value); },
+        $details,
+        array_keys($details)
```

Comment:
@angaither changed it to use `json_encode` instead which looks a lot cleaner and is easier to read! However, the message has some extra `\`s like so: `Request made. {"method":"POST","Transaction ID":"1476974774.215-0","Path":"http:\/\/rogue.dev:8000\/api\/v1\/reportbacks"}` - is this ok?